lib/core/src/sdk.rs

```diff
@@ -851,13 +857,18 @@ impl LiquidSdk {
         &self,
         amount_sat: u64,
         address: &str,
+        asset_id: &str,
     ) -> Result<u64, PaymentError> {
-        match self.estimate_onchain_tx_fee(amount_sat, address).await {
+        match self
+            .estimate_onchain_tx_fee(amount_sat, address, asset_id)
+            .await
+        {
             Ok(fees_sat) => Ok(fees_sat),
-            Err(PaymentError::InsufficientFunds) => self
-                .estimate_drain_tx_fee(Some(amount_sat), Some(address))
-                .await
-                .map_err(|_| PaymentError::InsufficientFunds),
+            Err(PaymentError::InsufficientFunds) if asset_id.eq(LIQUID_BTC_ASSET_ID.as_str()) => {
```

Comment:
Does this mean we don't want to support drain for non btc asset?

Reply:
It is not supported in that way by LWK, you should always be able to send all assets as long as you have the btc fees to do so. I will test to make sure this is the case